charts/pgadmin4/templates/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "pgadmin.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-        {{- with .Values.podLabels }}
+        {{- with (merge .Values.podLabels .Values.commonLabels) }}
```

Comment:
where is the podLabels for now?

Reply:
I don't want to delete the podLabels value (existing in the chart before), so I merge it with commonLabels
podLabels has the precedence over commonLabels